ci-automation/test.sh

```diff
@@ -118,54 +118,43 @@ function test_run() {
     local docker_vernum
     docker_vernum="$(vernum_to_docker_image_version "${vernum}")"
 
-    # Get SDK from either the registry or import from build cache
-    local sdk_version="${FLATCAR_SDK_VERSION}"
-    local sdk_name="flatcar-sdk-${arch}"
-    local docker_sdk_vernum="$(vernum_to_docker_image_version "${sdk_version}")"
-
-    docker_image_from_registry_or_buildcache "${sdk_name}" "${docker_sdk_vernum}"
-    local sdk_image="$(docker_image_fullname "${sdk_name}" "${docker_sdk_vernum}")"
-    echo "docker image rm -f '${sdk_image}'" >> ./ci-cleanup.sh
-
     local work_dir="__TESTS__"
     local tests_dir="${work_dir}/${image}"
     mkdir -p "${tests_dir}"
 
     local container_name="flatcar-tests-${arch}-${docker_vernum}-${image}"
+    local mantle_ref
+    mantle_ref=$(cat sdk_container/.repo/manifests/mantle-container)
 
     # Make the torcx artifacts available to test implementation
     __prepare_torcx "${arch}" "${vernum}" "${work_dir}"
 
-    # Ensure we run tests with an up-to-date kola.
-    # This covers the case of a mantle/kola commit ID in the ebuild being
-    #  newer than the mantle/kola included in the SDK.
-    ./run_sdk_container -x ./ci-cleanup.sh \
-        -n "${container_name}" -C "${sdk_image}" -v "${vernum}" \
-        sudo emerge mantle
-
     local tap_merged_summary="results-${image}.tap"
     local tap_merged_detailed="results-${image}-detailed.tap"
     local retry=""
     local success=false
+    # Also prune images to remove old mantle images
+    echo "docker rm -f '${container_name}' ; docker image prune -a -f --filter until=24h" >> ./ci-cleanup.sh
```

Comment:
We have the `docker-prune` service running as a cron job on all nodes that does this:
```
ExecStart=/usr/bin/docker system prune -fa --filter=until=12h
```
so I would leave only the `docker rm -f`